src/xrGame/GamePersistent.cpp

```diff
@@ -938,6 +941,22 @@ void CGamePersistent::LoadTitle(bool change_tip, shared_str map_name)
 
         pApp->LoadTitleInt(
             StringTable().translate("ls_header").c_str(), tmp.c_str(), StringTable().translate(buff).c_str());
+
+        if (g_pGameLevel)
+        {
+            const char* temp_map_name;
+            if (map_name == "zaton")
+                temp_map_name = "Затон";
+            else if (map_name == "jupiter")
+                temp_map_name = "Окрестности «Юпитера»";
+            else if (map_name == "jupiter_underground")
+                temp_map_name = "Путепровод «Припять-1»";
+            else if (map_name == "pripyat")
+                temp_map_name = "Припять";
+            else if (map_name == "labx8")
+                temp_map_name = "Лаборатория X8";
+            g_DiscordRPC->Update(temp_map_name);
```

Comment:
Again, that's fine for the first version. But I'd like to see these things in config files with localization. I'll leave priority of this for @Xottab-DUTY .